source/_integrations/apsystems.markdown

```diff
@@ -0,0 +1,22 @@
+---
+title: APsystems EZ1
+description: Control and monitor your APsystems EZ1 microinverters locally without the cloud
+featured: false
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_release: 2024.6
+ha_category:
+  - Energy
+ha_domain: apsystems
+ha_platforms:
+  - sensor
+ha_integration_type: device
+---
+
```

Comment:
Can you add some more information on what kind of data is shown? kostal plenticore, for example, provides a list of sensors.